src/test/resources/features/KickoutJourneys.feature

```diff
@@ -42,11 +42,23 @@ Feature: Kickout Journeys
     And I select "Yes" and continue in Check the name and address page for employee of org
     And I enter Name- "Automation Test" Email- "TestSaveAsDraft@automation.com",Phone- "9876543211", Job title- "employee of org" details
     When I click on cancel application link
-    And I click on confirm button on cancellation page
+    And I select "Yes" and continue on cancellation page
     And I click back button in the browser
     Then You cannot view this application page is displayed
     And I click on go to application and rulings page button
 
+  Scenario: User clicks on cancel application link with no value
+    Given I am on the ARS Home Page with affinity group as a "Individual" and Credential role as a "User"
+    And I click on Start new application in ARS Home
+    And I select role as a "An employee of the organisation"
+    And I click continue on Information you need to complete an application page
+    And I select "Yes" and continue in Are you planning to import goods page
+    And I click on continue in How We Contact You page
+    And I select "Yes" and continue in Check the name and address page for employee of org
+    And I enter Name- "Automation Test" Email- "TestSaveAsDraft@automation.com",Phone- "9876543211", Job title- "employee of org" details
+    When I click on cancel application link
+    And I select "No" and continue on cancellation page
+
```

Comment:
Should we have something like "Then x page is displayed"? When the user selects that they don't want to cancel their application, where should the user go next?